src/algebra/homology/homological_complex.lean

```diff
@@ -401,6 +401,37 @@ end
 namespace hom
 
 variables {C₁ C₂ C₃ : homological_complex V c}
+
+/-- The `i`-th component of an isomorphism of chain complexes. -/
+@[simps]
+def iso_app (f : C₁ ≅ C₂) (i : ι) : C₁.X i ≅ C₂.X i :=
+{ hom := f.hom.f i,
+  inv := f.inv.f i,
+  hom_inv_id' := by { erw [← comp_f, f.hom_inv_id, id_f] },
+  inv_hom_id' := by { erw [← comp_f, f.inv_hom_id, id_f] } }
```

Comment:
This can be simplified using `(homological_complex.eval_at _).map_iso f`. 
I think either way should be fine.